docs/understanding-the-flutterflow-ui/projects/how-to-run-test-projects.md

```diff
@@ -0,0 +1,94 @@
+---
+slug: how-to-run-test-projects
+title: How to Run and Test Projects
+tags: []
+toc_max_heading_level: 5
+---
+There are 4 ways to test your project in FlutterFlow.
+- Preview
+- Test
+- Run
+- Local Run
+
+## Preview Mode
+You can use the Preview Mode to quickly try out your app on a virtual device without waiting for it to build. This is helpful primarily for navigation and animations. You can also preview your app in the Dark/Light mode and visualize it on various mobile, tablet, and desktop devices.
+
+<div class="alert alert--info" role="alert">
+#### When to Use Preview Mode
+The main benefit is that your app will load instantly. However, because of the limitations this feature is used much less often that the other testing modes.
```

Comment:
I think we should mention about Preview Mode is just for UI testing, and doesn't load business logic as far as I remember and is meant to only "preview" the UI. So technically its not "building" the complete app. Can we please revise the description?